docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-workbench-tree-con.adoc

```diff
@@ -0,0 +1,13 @@
+[id='building-custom-dashboard-widgets-workbench-tree-con']
+= Navigation tree
+
+The workbench is the web-based application that combines all the web-based tools of {PRODUCT} into one configurable solution. It covers the complete life cycle of a {PRODUCT} project, beginning with the authoring phase, then on to the implementation and execution phase, and finally ending at monitoring phase.
+
+Workbench tree is the default navigation tree hierarchy that is displayed in the *Navigation* tab on the left of the *Page Editor* tool. You cannot delete the *Workbench* tree from the *Navigation* tab but you can edit the *Workbench* tree hierarchy to suit your requirements. You can add new user created groups as well as delete the existing groups from the *Workbench* tree. Any change to the *Workbench* tree causes the *Menu* on the *Home* screen to modify its entries. This feature helps users customize the workbench with new pages.
+
```

Comment:
The "Workbench" navigation tree contains the entries displayed in the workbench's mega menu bar. Changes to this navigation tree structure will be reflected in the mega menu. Such mechanism can be used, for example, to publish new pages.

Also, additional navigation trees can be created. Those custom trees can be used during a page edition to configure the navigation components within the page.

Reply:
Added the info as suggested by David. Rephrased the content to conform to guidelines.